.env.example

```
@@ -1,26 +1,29 @@
+POSTGRES_DB=festie
 
-# JWT
-JWT_SECRET=
+# event-service가 config-server yaml의 ${DB_USER} / ${DB_PASSWORD} 플레이스홀더 해소에 사용
+DB_USER=festie
+DB_PASSWORD=1234
 
-# Kafka
-KAFKA_BOOTSTRAP_SERVERS=
+# ============ Config Server ============
+CONFIG_USERNAME=festie          # 팀 공유값
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)